bidwire/scrapers/massgov_eopss_scraper.py

```diff
@@ -0,0 +1,60 @@
+from document import Document, get_new_urls
+from .base_scraper import BaseScraper
+from .massgov import url_scraper_dict
+from .massgov import results_page_scraper
+from db import Session
+import logging
+import scrapelib
+
+# Logger object for this module
+log = logging.getLogger(__name__)
+
+DOC_URL_PREFIX = 'http://www.mass.gov'
```

Comment:
It looks like this constant is only used once, to build `URL_PREFIX` below. Not sure it needs to be its own constant.